macros/schema_tests/at_least_one.sql

```diff
@@ -1,4 +1,8 @@
 {% macro test_at_least_one(model) %}
+  {{ adapter.dispatch('test_at_least_one', packages = dbt_utils._get_utils_namespaces())(model) }}
```

Comment:
This or something like it:
```suggestion
  {{ adapter.dispatch('test_at_least_one', packages = dbt_utils._get_utils_namespaces())(model, **kwargs) }}
```